framework/contracts/native/ibc-host/tests/integration.rs

```diff
@@ -87,6 +105,88 @@ fn account_creation() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[test]
+fn account_creation_full() -> anyhow::Result<()> {
+    let sender = Addr::unchecked("sender");
+    let chain = Mock::new(&sender);
+
+    let admin = Addr::unchecked("admin");
+    let mut admin_chain = chain.clone();
+    admin_chain.set_sender(admin.clone());
+
+    let admin_abstr = Abstract::deploy_on(admin_chain.clone(), admin.to_string())?;
```

Comment:
`admin_abstr` has confused me and @AndrewPochapsky. Can we call it `home_abstr` and the chain `home_chain`?

Reply:
Agree on this, updated!